static/js/vue-cdr-access/tests/unit/modalMetadata.spec.js

```diff
@@ -9,7 +10,7 @@ localVue.use(VueRouter);
 
 const updated_uuid = 'c03a4bd7-25f4-4a6c-a68d-fedc4251b680';
 const title = 'Test Collection';
-const response = `<table><tbody><tr><th>Creator</th> <td><p>Real Dean</p></td></tr></tbody></table>`;
+const response = pretty(`<table><tbody><tr><th>Creator</th><td><p>Real Dean</p></td></tr></tbody></table>`);
```

Comment:
Real dean is getting prettified!

Reply:
Yeah, vue test utils changed how they return HTML. It used to have no spaces, but they switched to formatting the HTML with the pretty package before returning it.